src/game.cpp

```diff
@@ -3119,22 +3120,32 @@ void Game::toggleFog(float *statustext_time, bool *flag)
 
 
 void Game::toggleDebug(float *statustext_time, bool *show_debug,
-		bool *show_profiler_graph)
+		bool *show_profiler_graph, bool *show_wireframe)
 {
 	// Initial / 3x toggle: Chat only
 	// 1x toggle: Debug text with chat
 	// 2x toggle: Debug text with profiler graph
+	// 3x toggle: Debug text with profiler graph and wireframe
```

Comment:
The profiler graph will obscure the wireframe graphics, i think the 3x toggle should be wireframe only.